en/news/_posts/2020-10-02-ruby-2-7-2-released.md

```diff
@@ -9,14 +9,14 @@ lang: en
 
 Ruby 2.7.2 has been released.
 
-This release contains intentional incompatibility. The deprecated warnings are off by default on 2.7.2 and later.
-You can turn on the deprecated warnings by specifing command line option -w or -W:deprecated.
+This release contains intentional incompatibility. Deprecation warnings are off by default on 2.7.2 and later.
+You can turn on deprecation warnings by specifying the -w or -W:deprecated option at the command-line.
 Please check the topics below for details.
 
 * [Feature #17000 2.7.2 turns off deprecation warnings by default](https://bugs.ruby-lang.org/issues/17000)
 * [Feature #16345 Don't emit deprecation warnings by default.](https://bugs.ruby-lang.org/issues/16345)
 
-This release contains the new version of webrick with a security fix described in the article.
+This release contains the new version of WEBrick with a security fix described in the article.
```

Comment:
"webrick" is the name of the gem. I think it's natural to use gem name in this context.